packages/manager/modules/pci/src/projects/project/public-gateways/add/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,39 @@
+{
+  "pci_projects_project_public_gateways_add_title": "Créer une instance Public Gateway",
+  "pci_projects_project_public_gateways_add_description1": "Créez une nouvelle instance Public Gateway pour accéder en toute sécurité aux réseaux externes.",
+  "pci_projects_project_public_gateways_add_description2": "Il s’agit du meilleur choix pour accéder à des ressources internet à partir d’instances privées (SNAT).",
+  "pci_projects_project_public_gateways_add_description3": "Vous pouvez également l'utiliser avec d'autres fonctions réseau, comme l’OVHcloud Load Balancer pour Public Cloud ou les IP flottantes. <a href=\"{{url}}\" target='_blank'>En savoir plus →</a>",
```

Comment:
**nitpick:** `En savoir plus` can be  put in a different translation key and use it along with `<a>` in HTML. Also use icons to display arrow. It must not be put in the translation file.